portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,47 +32,36 @@
 
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private ArrayList<String> comments;
-
-
-  @Override
-  public void init(){
-    comments = new ArrayList<String>();    
-  }
-
+  private ArrayList<String> comments;    
```

Comment:
no need to declare this as a member variable- can declare and initialize in the doGet function

Reply:
Thanks for the reminder! Totally forgot to change that